src/main/java/ca/on/oicr/gsi/dimsum/data/CaseData.java

```diff
@@ -23,10 +23,12 @@ public class CaseData {
   private final Set<String> requisitionNames;
   private final Set<String> projectsNames;
   private final Set<String> donorNames;
+  private final Map<String, ProjectSummary> projectsBySummary;
```

Comment:
Rename `projectSummariesByName` (values are `ProjectSummary`, key is the project name)